Index: src/main/resources/application.properties
IDEA additional info:
Subsystem: com.intellij.openapi.diff.impl.patch.CharsetEP
<+>ISO-8859-1
===================================================================
diff --git a/src/main/resources/application.properties b/src/main/resources/application.properties
new file mode 100644
--- /dev/null	(date 1694503371345)
+++ b/src/main/resources/application.properties	(date 1694503371345)
@@ -0,0 +1,19 @@
+spring.config.import=env.properties
+
+# database
+spring.datasource.url=jdbc:mysql://${DB_ADDRESS}/${DB_NAME}?serverTimezone=UTC&useUniCode=yes&characterEncoding=UTF-8
+spring.datasource.hikari.driver-class-name=com.mysql.cj.jdbc.Driver
+spring.datasource.hikari.username=${DB_USER}
+spring.datasource.hikari.password=${DB_PASSWORD}
+
+# jpa
+spring.jpa.hibernate.ddl-auto=none
+spring.jpa.show-sql=true
+spring.jpa.properties.hibernate.format_sql=true
+
+# logging
+logging.level.com.ssafy.recipe=debug
+
+# Eureka
+spring.application.name=${DOMAIN_NAME}
+eureka.client.service-url.defaultZone=${EUREKA_URL}
\ No newline at end of file
Index: .gitignore
IDEA additional info:
Subsystem: com.intellij.openapi.diff.impl.patch.BaseRevisionTextPatchEP
<+>HELP.md\r\n.gradle\r\nbuild/\r\n/src/main/resources/env.properties\r\n/src/main/resources/application.properties\r\n!gradle/wrapper/gradle-wrapper.jar\r\n!**/src/main/**/build/\r\n!**/src/test/**/build/\r\n\r\n### STS ###\r\n.apt_generated\r\n.classpath\r\n.factorypath\r\n.project\r\n.settings\r\n.springBeans\r\n.sts4-cache\r\nbin/\r\n!**/src/main/**/bin/\r\n!**/src/test/**/bin/\r\n\r\n### IntelliJ IDEA ###\r\n.idea\r\n*.iws\r\n*.iml\r\n*.ipr\r\nout/\r\n!**/src/main/**/out/\r\n!**/src/test/**/out/\r\n\r\n### NetBeans ###\r\n/nbproject/private/\r\n/nbbuild/\r\n/dist/\r\n/nbdist/\r\n/.nb-gradle/\r\n\r\n### VS Code ###\r\n.vscode/\r\n
Subsystem: com.intellij.openapi.diff.impl.patch.CharsetEP
<+>UTF-8
===================================================================
diff --git a/.gitignore b/.gitignore
--- a/.gitignore	(revision e57d42bc1c8194fe3bc6ce03bf9bd61e418c5a93)
+++ b/.gitignore	(date 1694503371345)
@@ -2,7 +2,7 @@
 .gradle
 build/
 /src/main/resources/env.properties
-/src/main/resources/application.properties
+
 !gradle/wrapper/gradle-wrapper.jar
 !**/src/main/**/build/
 !**/src/test/**/build/
